Node/index.html

```diff
@@ -0,0 +1,304 @@
+                            <div class="slide-img slide-img-1">
+                                <div class="item-img">
+                                    <img src="../img/image3.jpg" alt="">
+                                </div>
+                                <div class="slide-content">
+                                    <h4>Lorem ipsum dolor </h4>
+                                    <p>
+                                        Dignissimos! Nulla ex voluptates dolores mi ex. Doloribus, tempora commodo inceptos mollitia quos! Purus mollit reiciendis, mi exercitationem do nostrum dolor maiores orci nibh!
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="slide-img slide-img-2">
+                                <div class="item-img">
+                                    <img src="../img/image4.jpg" alt="">
+                                </div>
+                                <div class="slide-content">
+                                    <h4>Lorem ipsum dolor </h4>
+                                    <p>
+                                        Dignissimos! Nulla ex voluptates dolores mi ex. Doloribus, tempora commodo inceptos mollitia quos! Purus mollit reiciendis, mi exercitationem do nostrum dolor maiores orci nibh!
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="slide-img slide-img-3">
+                                <div class="item-img">
+                                    <img src="../img/image2.jpg" alt="">
+                                </div>
+                                <div class="slide-content">
+                                    <h4>Lorem ipsum dolor </h4>
+                                    <p>
+                                        Dignissimos! Nulla ex voluptates dolores mi ex. Doloribus, tempora commodo inceptos mollitia quos! Purus mollit reiciendis, mi exercitationem do nostrum dolor maiores orci nibh!
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                        <a class="col-md-2 right control" href="#myCarousel" data-slide="next">
+                            <i class="fa fa-chevron-right" aria-hidden="true"></i>
+                        </a>
+                    </div>
+                </div>
+            </div>
+            <div class="container container-4">
+                <h3> sed tempus aliqliam</h3>
+                <p>quis nostrud exercitation ullamco laboris</p>
+                <div class="container-4-box">
+                    <div class="row container-4-row row-text-1">
+                        <div class="col-md-5 col-text">
+                            <div class="col-content">
+                                <h5>Lorem ipsum </h5>
```

Comment:
mục đích có dấu cách ở đấy là gì?
```suggestion
                                <h5>Lorem ipsum</h5>
```